network_simulator/statistics.py

```diff
@@ -181,7 +180,18 @@ def flow_packet_received(self, flow, ack_packet, curr_time):
         # Retrieve data packet sent time from the ack packet and use
         # it to calculate RTT.
         sent_time = ack_packet.data_packet_start_time_sec
-        stats.packet_rtts.append(curr_time - sent_time)
+        stats.packet_rtts.append(
```

Comment:
packet_rtts is a list of (timestamp, RTT) tuples, so what you want to record is (curr_time, curr_time - sent_time)